src/Silex/EventListener/MiddlewareListener.php

```diff
@@ -57,7 +64,7 @@ public function onKernelRequest(GetResponseEvent $event)
                 $event->setResponse($ret);
 
                 return;
-            } elseif (null !== $ret) {
+            } elseif ($ret !== null) {
```

Comment:
ret is a pretty bad variable name (super minor and I know symfony uses it internally everywhere)